core/src/main/resources/hudson/logging/Messages.properties

```
@@ -21,4 +21,5 @@
 # THE SOFTWARE.
 
 LogRecorderManager.init=Initialing log recorders
-LogRecorderManager.DisplayName=System Log
\ No newline at end of file
+LogRecorderManager.DisplayName=System Log
+LogRecorder.Target.Empty.Warning=Setting the Root (empty) logger to FINE or below is strongly discouraged as it excessively impairs performances. It may well crash Jenkins and prevent it to start.
```

Optional alternate phrasing
```suggestion
LogRecorder.Target.Empty.Warning=Setting the Root (empty) logger to FINE or below is strongly discouraged. It may critically impact performance. It may crash Jenkins and prevent Jenkins from starting.
```